feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/BookmarkIcon.kt

```diff
@@ -0,0 +1,62 @@
+package io.github.droidkaigi.confsched2023.sessions.component
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.BookmarkBorder
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.semantics.onClick
+import androidx.compose.ui.semantics.semantics
+import com.airbnb.lottie.compose.LottieAnimation
+import com.airbnb.lottie.compose.LottieCompositionSpec.RawRes
+import com.airbnb.lottie.compose.animateLottieCompositionAsState
+import com.airbnb.lottie.compose.rememberLottieComposition
+import io.github.droidkaigi.confsched2023.feature.sessions.R
+
+@Composable
+fun BookmarkIcon(
+    isBookmarked: Boolean,
+    contentDescription: String,
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    var isPlaying by remember { mutableStateOf(false) }
+    val lottieComposition by rememberLottieComposition(RawRes(R.raw.add_to_bookmark_lottie))
+    val progress by animateLottieCompositionAsState(
+        composition = lottieComposition,
+        isPlaying = isPlaying,
+    )
+
+    Box(
+        modifier = modifier
+            .clickable {
+                isPlaying = isBookmarked.not()
+                onClick()
+            },
+        contentAlignment = Alignment.Center,
+    ) {
+        if (isBookmarked) {
+            LottieAnimation(
```

Comment:
@Corvus400 
If the animation completely finished; progress >= 100, I think a plain icon composable with `Icons.Filled.Bookmark` should be rendered instead of LottieAnimation. 

Another solution is to modify the lottie animation to show bookmark icons properly before/after the animation frames.

|Progress|Visual|
|:---|:----|
|0|not bookmarked|
|1-99|bookmark animation|
|100|bookmarked|

Reply:
I see, thank you very much!
I think it would be better to have the 0th frame of the Lottie file corrected.
Is it possible to fix it?🙇‍♂️

Reply:
@Corvus400 
Sure thing and let me clarify one thing before requesting this stuff to our designers.

Let's say we have a session has already been bookmarked before opening this screen. And then, when we open this screen with *clean* composable states, this composable must skip an animation somehow to prevent a weird behaviour. I'm not sure if it's easy to skip an animation and can show the last frame naturally. I'm afraid it would be a new puzzle. Do you know it?

Reply:
@jmatsu 
> I'm afraid it would be a new puzzle. Do you know it?

I see...simply modifying the Lottie file would create a new problem.🤔
I have tried to formulate the idea once I received [it here](https://github.com/DroidKaigi/conference-app-2023/pull/936/commits/6f45dcc4aa59244684cf7046c7d3490a55af47c4).💁‍♂️
It seems to be working well, but there is one problem.
I can't seem to find the same icons in the Material Icons as in Figma. 🙇 
I may need to have it so that only this 0th frame image can be exported from Figma.

Reply:
@Corvus400 
I'm sorry for my misunderstanding. This animation should be applied to *the icon in the header* but not to *each bookmark icon of sessions*. Could you please check the video below and implement it? If you have any question, please feel free to ask us. Thanks.

https://github.com/DroidKaigi/conference-app-2023/assets/4340693/4f0e8bb8-56c2-4382-aac1-36d3da9489f0

Reply:
@jmatsu 
Ah I see... sorry, I finally realized now that I was mistaken. 🙇 
I'll have to re-implement it.

Reply:
No worries! 👌